client/src/pages/Edit.js

```diff
@@ -91,18 +99,43 @@ function Edit() {
               />
             </div>
             <div className="container text-center">
-              <Button variant="contained" type="submit">
-                Updated Item
-              </Button>
-              <Button
-                variant="contained"
-                type="submit"
-                onClick={navigateToEdit}
-              >
-                Return To Details Page
-              </Button>
+              {edited ? (
+                <>
+                  <Button
+                    variant="contained"
+                    type="submit"
+                    onClick={() => setEdited(!edited)}
+                    onClick={notify}
+                  >
+                    Updated Item
+                  </Button>
+                  <ToastContainer />
+                  <div>
+                    <Button
+                      variant="contained"
+                      type="submit"
+                      onClick={navigateToEdit}
+                    >
+                      Return To Details Page
+                    </Button>
+                  </div>
+                </>
+              ) : null}
+              {!edited ? (
+                <div>
+                  <h1>Error Message</h1>
+                </div>
+              ) : null}
```

Comment:
Can the second ternary here be used in the `else` branch of the first instead of using two ternary conditions here?

```suggestion
              {edited ? (
                <>
                  <Button
                    variant="contained"
                    type="submit"
                    onClick={() => setEdited(!edited)}
                    onClick={notify}
                  >
                    Updated Item
                  </Button>
                  <ToastContainer />
                  <div>
                    <Button
                      variant="contained"
                      type="submit"
                      onClick={navigateToEdit}
                    >
                      Return To Details Page
                    </Button>
                  </div>
                </>
              ) : (
                <div>
                  <h1>Error Message</h1>
                </div>
              )}
```